release-notes/rn-axon-framework/rn-af-minor-releases.md

```diff
@@ -6,61 +6,61 @@ Any patch release made for an Axon project is tailored towards resolving bugs. T
 
 ### Release 4.3.5
 
- * The `TrackingEventProcessor#mergeSegment(int)` method was invoked with the high segment number of the pair to merge,
- 
-    an error would occur in the process as it expected to receive the lower number on all scenarios.
-    
-   This was resolved in pull request [#1450](https://github.com/AxonFramework/AxonFramework/pull/1450).
-   
- * A small connectivity adjustment which was performed in the `AxonServerConnectionManager` for bug release 4.3.4 has been reverted.
- 
-   Although it worked successfully for some scenarios, it did not correctly cover all possibilities.
-   
-   The commit can be found [here](https://github.com/AxonFramework/AxonFramework/commit/5b9348040f4f977db3b9a15c3ae55904710814b6) for reference.
-   
-   The full scenario will be covered through the adjusted connector which is underway for beta release in 4.4.
+* The `TrackingEventProcessor#mergeSegment(int)` method was invoked with the high segment number of the pair to merge,
```

Comment:
Would be interested to know what tool you use for adjusting the indentation here and adding the backslashes to the issue numbers. Asking this as I recently added these release notes and would like to maintain a consistent styling throughout our reference guide.

Reply:
Regular gitbook utility